### PR TITLE
Create method for insert a new project (controller+repository+service…

### PR DESCRIPTION
**Pull request by iancdiana21**
…+model) + add authorisation in Filter + JUnit Test

### CODE REVIEW

**Comment by iancdiana21** [comment]
[[Admin] BE Insert new Project](https://trello.com/c/VRSuHV7w/63-admin-be-insert-new-project)

File: src/main/java/ro/db/vendor/model/ProjectModel_ForInsertNewProject.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package ro.db.vendor.model;
+
+public class ProjectModel_ForInsertNewProject {
```
[the remark this answers is not in the document]
Comment (o-dan) [reply src/main/java/ro/db/vendor/model/ProjectModel_ForInsertNewProject.java]: Please, have a look here: http://www.oracle.com/technetwork/java/codeconventions-135099.html

[the remark this answers is not in the document]
Comment (iancdiana21) [reply src/main/java/ro/db/vendor/model/ProjectModel_ForInsertNewProject.java]: I read the code conventions and I changed the class name... I'm sorry for that mistake.

[the remark this answers is not in the document]
Comment (alexandrubirlan) [reply src/main/java/ro/db/vendor/model/ProjectModel_ForInsertNewProject.java]: No worries. It's but about mistaking it being sorry

**Pull request merged**